.github/workflows/master-mirrors-main.yaml

```
@@ -0,0 +1,19 @@
+name: 'master mirrors this branch'
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  reattach-master:
```

nit and non-blocker: `sync-master-main`?

---

I have an alternative idea (because the actions UI already shows "sync master with main" so saying the same thing twice would be redundant.
```suggestion
  push-master:
```